windows/CodePush/CodePush.vcxproj

```diff
@@ -57,7 +57,7 @@
   </ItemGroup>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '14.0'">v140</PlatformToolset>
```

Comment:
vs14 is out of support. If it only available for vs14, then let's remove this line. But how did it work with vs19 without this condition?